.github/workflows/ci-testing.yml

```diff
@@ -63,6 +63,11 @@ jobs:
         brew update
         brew install libomp  # https://github.com/pytorch/pytorch/issues/20030
 
+    - name: Install graphviz
+      if: matrix.topic == 'serve'
```

Comment:
I would install it on all Linux as we spoke
and for other OS add pytest argument `--ignore tests/serve`

Reply:
But we already have a seperate run for serve dependencies which is only on ubuntu, so it makes more sense to just install graphviz there right?

Reply:
oh, yes, I see https://github.com/PyTorchLightning/lightning-flash/blob/01fef90d34df2065b65684b7ddc2e7780d12ccfa/.github/workflows/ci-testing.yml#L47